domains/janco.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "JancoNel",
+           "email": "",
+           "discord": "1264555376918134796"
+        },
+    
+        "record": {
+            "CNAME": "https://janconel.github.io/personalwebsite/"
```

```suggestion
            "CNAME": "https://janconel.github.io"
```
CNAME records cannot contain slashes (`/`).